src/test/validation_block_tests.cpp

```diff
@@ -23,7 +23,7 @@ BOOST_FIXTURE_TEST_SUITE(validation_block_tests, RegtestingSetup)
 struct TestSubscriber : public CValidationInterface {
     uint256 m_expected_tip;
 
-    TestSubscriber(uint256 tip) : m_expected_tip(tip) {}
+    explicit TestSubscriber(uint256 tip) : m_expected_tip(tip) {}
```

Comment:
13769 https://github.com/bitcoin/bitcoin/pull/13769/files#diff-b12066a688b85bea440eb38f079fb8ecc1984318470631bca7e3547a98effaa9L32-R32 is missing. HasReason is in test_dash.h

Reply:
Brought in missing changes